src/target/llvm/intrin_rule_rocm.cc

```diff
@@ -40,8 +41,59 @@ inline void DispatchExternOCML(const TVMArgs& args, TVMRetValue* rv) {
   *rv = CallNode::make(call->dtype, intrinsic_name.str(), call->args, CallNode::PureExtern);
 }
 
+inline void DispatchShuffle(const TVMArgs& targs, TVMRetValue* rv) {
+  PrimExpr e = targs[0];
```

Comment:
Let's use more descriptive names for all arguments.

Reply:
While I do agree in general, the e here is more or less a cast and I took this straight from `intrin_rule_llvm.h`, including the naming.